index.js

```diff
@@ -417,6 +422,39 @@ function getPullRequestToMerge () {
 
 }
 
+function listPullRequestsToMerge() {
+	config.pullRequests.forEach(function(pullRequest, index){
+		var ticket = parseTicketFromPullRequest(pullRequest.head.ref);
+		var notice;
+		if (ticket){
```

Comment:
Actually, I do understand now, but this is too tightly associated to the BE process, not sure I want to do that since the other repositories are not supporting that.